src/projects/republic/config/index.tsx

```diff
@@ -74,4 +70,6 @@ export const republicConfig: ProjectConfig = merge({}, defaultConfig, {
     showTopMenuButton: true,
     showWindowSideBar: true,
   },
+  // TODO: remove next dev line
+  showMirador: false,
```

Comment:
back to true

Reply:
Fixed in [7e5aa03](https://github.com/knaw-huc/textannoviz/pull/132/commits/7e5aa03f10b7fde07fceec7fb2ce25154e492f5c)